README.md

```diff
@@ -8,6 +8,10 @@
 - 英文名：GMT China Community
 - 主页：https://gmt-china.org
 - GitHub 组织：https://github.com/gmt-china
+  - [所有者](https://github.com/orgs/gmt-china/people?query=role%3Aowner)：
+    - [@core-man](https://github.com/core-man)
+    - [@seisman](https://github.com/seisman)
+    - [@wangliang1989](https://github.com/orgs/gmt-china/people/wangliang1989)
```

Comment:
```suggestion
    - [@seisman](https://github.com/seisman)
```

Reply:
I cannot make the changes. See what I see:
![](https://user-images.githubusercontent.com/50591376/123650628-53895780-d85d-11eb-83cc-99a35d486678.png)